nntrainer/layers/pooling2d_layer.h

```diff
@@ -37,17 +40,26 @@ class Pooling2DLayer : public LayerV1 {
     unknown = 4,
   };
 
+  /**
+   * @brief PaddingType Class
+   * @todo support keras type of padding
+   */
+  enum class PaddingType {
```

Comment:
Could you delete this? I won't in #1360

Reply:
It is used in parse_util.cpp. 
I recommend you to delete it in #1360 as deleting also doesn't match the patch scope.

Reply:
I see, I'll do it after #1360, #1359 is merged.

Reply:
sure.